### PR TITLE
Update node version mismatch message

### PR DESCRIPTION
**Pull request by kuldeepsidhu88**
This fixes issue #1622

### CODE REVIEW

**Comment by facebook-github-bot-4** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: presto-main/src/main/java/com/facebook/presto/metadata/DiscoveryNodeManager.java [hunk 1]
```diff
@@ -181,7 +181,7 @@ private synchronized PrestoNode refreshNodesInternal()
                 // record current node
                 if (node.getNodeIdentifier().equals(nodeInfo.getNodeId())) {
                     currentNode = node;
-                    checkState(currentNode.getNodeVersion().equals(expectedNodeVersion), "INVARIANT: current node version should be equal to expected node version");
+                    checkState(currentNode.getNodeVersion().equals(expectedNodeVersion), "INVARIANT: current node version should be equal to "+expectedNodeVersion);
```
Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/metadata/DiscoveryNodeManager.java]: codestyle, no spaces around `+`

see the travis build

Comment (kuldeepsidhu88) [inline presto-main/src/main/java/com/facebook/presto/metadata/DiscoveryNodeManager.java]: Thanks. Updated!

**Comment by kokosing** [comment]
looks good

**Comment by facebook-github-bot-2** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by martint** [comment]
Merged with a couple of minor changes. Thanks!

**Pull request closed without merging**